src/components/actionBar/actionBar/ActionBar.tsx

```diff
@@ -0,0 +1,161 @@
+    const searchBusHandler = () => {
+        // api call to get listing data
+        // apply loading states
+    };
+
+    return (
+        <Wrapper>
+            <Stack spacing={5} direction="row">
+                <Stack spacing={1} width="1000px" direction="row">
+                    <Autocomplete
+                        fullWidth
+                        options={locationOptions.filter((loc) => {
+                            return loc != stopLocation;
+                        })}
+                        onChange={handleStartSelect}
+                        renderInput={(params) => (
+                            <TextField
+                                {...params}
+                                label="From"
+                                InputProps={{
+                                    ...params.InputProps,
+                                    startAdornment: (
+                                        <>
+                                            <InputAdornment position="start">
+                                                <PanoramaFishEyeSharp />
+                                            </InputAdornment>
+                                            {params.InputProps.startAdornment}
+                                        </>
+                                    ),
+                                }}
+                            />
+                        )}
+                    ></Autocomplete>
+                    <IconButton>
+                        <SwapHoriz />
+                    </IconButton>
+                    <Autocomplete
+                        fullWidth
+                        options={locationOptions.filter((loc) => {
+                            return loc != startLocation;
+                        })}
+                        onChange={handleStopSelect}
+                        renderInput={(params) => (
+                            <TextField
+                                {...params}
+                                label="To"
+                                InputProps={{
+                                    ...params.InputProps,
+                                    startAdornment: (
+                                        <>
+                                            <InputAdornment position="start">
+                                                <FmdGood />
+                                            </InputAdornment>
+                                            {params.InputProps.startAdornment}
+                                        </>
+                                    ),
+                                }}
+                            />
+                        )}
+                    ></Autocomplete>
+                </Stack>
+                <DatePicker
+                    label="Date"
+                    slots={{
+                        openPickerIcon: Today,
+                    }}
+                    slotProps={{
+                        // textField: {
```

Comment:
Do we need to keep these commented lines?

Reply:
No they are not necessary. Removing commented code.